DAO.sol

```diff
@@ -389,6 +389,9 @@ contract DAO is DAOInterface, Token, TokenSale {
             throw;
         }
 
+        if (_debatingPeriod > 8 weeks)
```

Comment:
because of https://github.com/slockit/DAO/issues/75
To prevent someone making a extreme long debating period, and people have blocked tokens forever,